### DISCUSSION TITLE
follow mode not working in history

### DISCUSSION THREAD

**Discussion by pbwn** [Jelmerro/Vieb#519] [General]
i have been using vieb for a long time but recently moved to arch (from debian based linux) and downloaded vieb directly from the site and installed with pacman -U. When i do ctrl h to view history i always immediately hit f to select the search field. On my new install the search field is not annotated with a letter to select it, and other urls in history aren't select-able in the same way. I never experienced this problem before. I wonder if anyone else has experienced this before i make a bug report. Many thanks for creating Vieb!

**Comment by Jelmerro** [comment]
Could you send a screenshot of the problem you are describing? Here is what it looks like for me.
![image](https://github.com/Jelmerro/Vieb/assets/1696674/8edb0f96-f3bf-412d-a0f4-c1870f828ba0)

**Reply by pbwn** [reply]
thanks, mine looks different, as follows, without the filter highlighted, nor the links below it
![Screenshot_2023-10-10_22-19-00](https://github.com/Jelmerro/Vieb/assets/34809091/5570f481-b7ad-4f86-a607-073ac5f50259)

**Reply by Jelmerro** [reply]
That looks very weird indeed. Do you have this on other pages, which colorscheme is this, do you have errors in the console and what does the `set followelement?` return?

**Reply by pbwn** [reply]
thanks. F and f work fine on all other pages. I use the verticaltabs color scheme. I copied my viebrc from my previous computer where I didn't experience this issue. Set followelement give the following:
![Screenshot_2023-10-11_13-08-04](https://github.com/Jelmerro/Vieb/assets/34809091/3e14006a-7fdb-492b-beb9-8b58b53988e1)
I see no errors in the console when viewing history and trying to use f.

**Reply by Jelmerro** [reply]
This is a bug in the `verticaltabs` colorscheme, though it is caused by duplicate ids on the history page, which I will fix by renaming it to `hist-container` instead of `page-container` (which is also used for pages in `#app`).
